app/src/main/java/com/github/bgabriel998/softwaredevproject/InitActivity.java

```diff
@@ -0,0 +1,39 @@
+package com.github.bgabriel998.softwaredevproject;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.View;
+import android.widget.ProgressBar;
+
+import com.github.giommok.softwaredevproject.FirebaseAccount;
+import com.google.firebase.FirebaseApp;
+
+public class InitActivity extends AppCompatActivity {
```

Comment:
Should we also request the POIs in this activity or should this be done later in the main activity?

Reply:
This will be done in an optimization Task I guess, since we don't use the POIs for the moment